src/lib/containers/StackedRowHomebrew.js

```diff
@@ -42,10 +50,82 @@ export default class StackedRowHomebrew extends React.Component {
     handleExit (event) {
         // remove box shadow
         event.target.style.boxShadow = ""
-         // careful to avoid an infinite loop
-        if (this.state.hoverText !== "") {
-            this.setState({hoverText: ""})
+    }
+
+    /**
+     * Handle column click event
+     */
+    handleClick = (dict) => (event) => {
+        // https://medium.freecodecamp.org/reactjs-pass-parameters-to-event-handlers-ca1f5c422b9
+        this.setState(
+            {
+                hoverText: dict.value,
+                hoverTextCount: dict.count,
+                index: dict.index
+            }
+        )
+    }
+
+    /**
+     * Update the state with selected facets and call props to update data
+     *
+     * @param {*} selectedFacets
+     * @memberof Facets
+     */
+    updateStateAndMakeQuery(selectedFacets) {
+        this.setState({ selectedFacets });
+        // have to reformat the selected facets to format for the api call
+        let selectedFacetsFormatted = Object.keys(selectedFacets).map(
+            key => {
+                return selectedFacets[key]
+            }
+        )
+        let queryRequest = this.props.getLastQueryRequest();
+        queryRequest.query.selectedFacets = selectedFacetsFormatted;
+        this.props.executeQueryRequest(queryRequest);
+    }
+
+    // Handle user cycling through slices of the bar chart
```

Comment:
this is fun, but it's a secret feature (so you can optionally remove for simplicity).